src/main/java/frc/robot/commands/autonomousCommands/AutonPathCommand.java

```diff
@@ -0,0 +1,76 @@
+package frc.robot.commands.autonomousCommands;
+
+import java.io.IOException;
+import java.nio.file.Path;
+
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.math.controller.RamseteController;
+import edu.wpi.first.math.controller.SimpleMotorFeedforward;
+import edu.wpi.first.math.trajectory.Trajectory;
+import edu.wpi.first.math.trajectory.TrajectoryConfig;
+import edu.wpi.first.math.trajectory.TrajectoryUtil;
+import edu.wpi.first.math.trajectory.constraint.DifferentialDriveVoltageConstraint;
+import edu.wpi.first.wpilibj.DriverStation;
+import edu.wpi.first.wpilibj.Filesystem;
+import edu.wpi.first.wpilibj2.command.RamseteCommand;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import frc.robot.constants.DriveConstants;
+import frc.robot.subsystems.drive.Drive;
+
+public class AutonPathCommand extends SequentialCommandGroup {
+
+private Drive drive; 
+private DifferentialDriveVoltageConstraint autoVoltageConstraint; 
+private TrajectoryConfig config;
+private RamseteCommand ramseteCommand;
+
+//values to rememeber for robot velocity= 3.66 - Acceleration= 1.83 (these are starting values that was changed over time)
+public final double kMaxSpeedMetersPerSecond = 0.90;
+public final double kMaxAccelerationMetersPerSecondSquared = 0.40;
+public final double MAX_VOLTAGE = 10;
```

Comment:
Let's make this adjustable via the constructor

Reply:
That way we can tune per path

Reply:
how would you go about doing that because when making pathweaver you set the max accel and velocity, so every path has that same setting unless you make a whole new project

Reply:
Can you add a screenshot to show me what you mean

Reply:
which also leaves the question, does pathweaver override these values? Values for pathweaver is 2.2 SpeedMPS and 0.6 AccelMPS squared

Reply:
Looking at the JSON the values aren't set there

Reply:
![image](https://user-images.githubusercontent.com/3326002/154557006-0dff4b47-dac0-4d20-9e93-24712252d645.png)

Reply:
So basically when we set up the command I want to do something like
`new PathCommand(json, maxAcc, maxVel, maxVoltage)`

Reply:
looking at the code I guess it doesn't anymore, only did that for tutorial

Reply:
right now all of the path commands are hard coded to have certain values

Reply:
> looking at the code I guess it doesn't anymore, only did that for tutorial

wym?

Reply:
![Capture](https://user-images.githubusercontent.com/89940965/154557656-eccc1ed2-f577-4478-80cd-cea941ebda80.PNG)

Reply:
this is what I was talking about

Reply:
Those values aren't reflected in the JSON output, if you look at the path json it's only a bunch of coordinates. So you're expected to set those via code. Does that make sense? 

So making them configurable like the paths makes sense because you want to set diff values per path

Reply:
in the wpilib.json file it shows accel 0.6 and the velocity on each file

Reply:
and other values being set on each path

Reply:
You're totally right. let me find out what these actually mean

Reply:
Ok looking at this: https://github.com/wpilibsuite/allwpilib/blob/main/wpimath/src/main/java/edu/wpi/first/math/trajectory/TrajectoryConfig.java#L39

It looks like it's just setting a maximum velo and acc. So incase you "fat finger" the values, they help prevent the robot to fly off the handle. I think it also does calculations between two points, and these values prevent the calculated numbers from being too high.

We can leave this as is and revisit later

Reply:
Fairly confident the points represent the goal robot pose at specific time steps. If this is the case, you would be able to adjust max velocity and max acceleration as a pair (by adjusting the time between steps), but adjusting velocity and acceleration independently would be very difficult.

Reply:
Actually let's go ahead and include them in the constructor